project/B10732009/README.md

```diff
@@ -0,0 +1,110 @@
+# Procedural Terrain Generation #
+
+Procedurally generate random terrain with [Perlin Noise](https://en.wikipedia.org/wiki/Perlin_noise).
+
+## Basic Information ##
+
+GitHub repository : [https://github.com/B10732009/procedural-terrain-generation](https://github.com/B10732009/procedural-terrain-generation)
+
+**[Procedural Generation](https://en.wikipedia.org/wiki/Procedural_generation)** is a method of **creating data algorithmically** as opposed to manually, typically through a combination of human-generated assets and algorithms coupled with computer-generated randomness and processing power. 
+
+In video games,  **Procedural Terrain Generation** is very useful where you want to generate natural terrain (caves, hills, rivers, etc.) that has a smooth feel, but is still random.
+
+## Problem to Solve ##
+
+**Perlin Noise** is a very popular algorithm for procedural generation developed by Ken Perlin in 1983. It can be used for any kind of wave-like, undulating material, texture, or terrain. 
+
+Compared with just some random values, Perlin Noise can generate values very smoothly and continuously, which looks more realistic in terrain generation. 
+
+The graph below shows the difference between normal random values and perlin noise.
+
+<p style="text-align: center;">
+    <img src="img/compare.png" alt="compare">
+</p>
+
+In this project, I am going to implement a set of API (for C++ and Python) of the Perlin Noise algorithm in 1D, 2D, and 3D (if time is enough), and use these API to render some terrain with [Ursina](https://www.ursinaengine.org/) (a game engine in Python).
```

Comment:
I am curious what will be the use case of 3D Perlin Noise.  But the proposal is good without the information.  Having it included in your final presentation would be good.